### PR TITLE
uartpb: define a unique output message type

### PR DESCRIPTION
**Pull request by ecourtois**
uartpb module can be used to send Protobuf messages over UART from any
level (lib, sys, platform, application).
Now, uartpb can send only one type of message, defined by the application,
containing a 'oneof' attribute (similar to 'union' in C). This attribute
must group all Protobuf types that can be sent over UART.

### CODE REVIEW

File: examples/uartpb/PB_Color.proto [hunk 1]
```diff
@@ -1,7 +1,5 @@
 syntax = "proto3";
```
Comment (gdoffe) [inline examples/uartpb/PB_Color.proto:1]: Commit message:
s/Emebbed/Embedded/

File: sys/uartpb/include/uartpb/UartProtobuf.hpp [hunk 2]
```diff
@@ -8,6 +8,11 @@
 /// @details     This module provides a class that controls an UART to send and
 ///              receive Protobuf messages using EmbbededProto.
 ///              A callback function must also be provided to handle decoded messages.
+///              A file `uartpb_config.hpp` be provided by the application, either at lib,
```
Comment (gdoffe) [inline sys/uartpb/include/uartpb/UartProtobuf.hpp:11]: must be ?

File: sys/shell_menu/Makefile.dep [hunk 3]
```diff
@@ -2,6 +2,5 @@ USEPKG += embedded-proto
```
Comment (gdoffe) [inline sys/shell_menu/Makefile.dep:2]: commit message:
s/is/if/

**Review by gdoffe — APPROVED** [review]
Just typo, I approve. Fix typos and merge please. ;)

File: examples/uartpb/PB_Color.proto (hunk 1 above)
Comment (ecourtois) [reply examples/uartpb/PB_Color.proto:1]: Fixed

**Pull request merged**